### ISSUE TITLE
No payment method available, GetRatesAsync() Unexpected exception

### ISSUE THREAD

**Issue by hakonamatata** [btcpayserver/btcpayserver#254]
Hi

I keep getting this after updating to the latest version. Some times everything works, some times I get this error. 

```
  | "No payment method available for this store\n8/24/18 9:29:43 AM: 
Creation of invoice starting\n8/24/18 9:29:43 AM: BTC: Unexpected exception (System.InvalidOperationException: The rate has expired\n
   at BTCPayServer.Services.Rates.BackgroundFetcherRateProvider.LatestFetch.GetResult() in /source/Services/Rates/BackgroundFetcherRateProvider.cs:line 30\n
   at BTCPayServer.Services.Rates.BackgroundFetcherRateProvider.GetRatesAsync() in /source/Services/Rates/BackgroundFetcherRateProvider.cs:line 84\n
   at BTCPayServer.Services.Rates.RateProviderFactory.QueryRates(String exchangeName) in /source/Services/Rates/RateProviderFactory.cs:line 165\n
   at BTCPayServer.Services.Rates.RateFetcher.GetRuleValue(List`1 dependentQueries, RateRule rateRule) in /source/Services/Rates/RateFetcher.cs:line 77\n
   at BTCPayServer.Controllers.InvoiceController.CreatePaymentMethodAsync(Dictionary`2 fetchingByCurrencyPair, IPaymentMethodHandler handler, ISupportedPaymentMethod supportedPaymentMethod, BTCPayNetwork network, InvoiceEntity entity, StoreData store, InvoiceLogs logs) in /source/Controllers/InvoiceController.cs:line 211)\n"
-- | --

```

**Comment by hakonamatata** [comment]
Not sure exactly why it started happening, but I did change the update frequency to 20 min.

**Comment by NicolasDorier** [comment]
confirmed this is bug, working on it.

**Comment by NicolasDorier** [comment]
Fix on 1.0.1.98
